flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -968,13 +967,14 @@ private void checkpointState(
 		checkpointingOperation.executeCheckpointing();
 	}
 
-	private void initializeState() throws Exception {
+	private void initializeStateAndOpen() throws Exception {
```

Comment:
The `openAllOperators()` method is now unused and should be removed. We should also move the comment that explains the order of initialization/opening from `openAllOperators()` to this method.